source/server/hot_restart_impl.cc

```diff
@@ -153,8 +170,8 @@ int HotRestartImpl::bindDomainSocket(uint64_t id, Api::OsSysCalls& os_sys_calls)
 }
 
 sockaddr_un HotRestartImpl::createDomainSocketAddress(uint64_t id) {
-  // Right now we only allow a maximum of 3 concurrent envoy processes to be running. When the third
```

Comment:
any reason to reformat these comments?

Reply:
I think clang-format hit this; not sure why it got reformatted.  I'll revert.